xbmc/video/dialogs/GUIDialogVideoSettings.cpp

```diff
@@ -178,7 +178,8 @@ void CGUIDialogVideoSettings::OnSettingChanged(const std::shared_ptr<const CSett
   else if (settingId == SETTING_VIDEO_TONEMAP_PARAM)
   {
     CVideoSettings vs = g_application.GetAppPlayer().GetVideoSettings();
-    vs.m_ToneMapParam = static_cast<float>(std::static_pointer_cast<const CSettingNumber>(setting)->GetValue());
+    float val = static_cast<float>(std::static_pointer_cast<const CSettingNumber>(setting)->GetValue());
+    vs.m_ToneMapParam = std::roundf(val * 100) / 100; // round to 2 decimal places
```

Comment:
What's the reason for this? The slider should only allow 0.1 increments as defined here: https://github.com/xbmc/xbmc/blob/b5871f5c48df16f862930773bddbd36dc2e13dc5/xbmc/video/dialogs/GUIDialogVideoSettings.cpp#L420-L422

min = 0.1
step = 0.1
max = 5.0

Reply:
Without this inexacte values like 0.9988346789 are stored on xml. 

In fact, it is no longer possible to save the default value of 1.00000 anymore when slider has moved.

`<tonemapparam>1.000000</tonemapparam>`

Reply:
Also on screen step is not 0.1 but 0.01

![Captura de pantalla 2020-11-11 103424](https://user-images.githubusercontent.com/58434170/98794704-a4048f80-2409-11eb-8abc-5816eff2ceb1.png)

Reply:
Don't know how you managed that as it's 0.10 steps here https://youtu.be/s_1Mbeg836o

Reply:
maybe you need to reset your settings?

Reply:
Video is moving slider with keyboard... try with mouse... 
Anyway the problem is not number in screen (only 2 decimals is shown) the problem is internal value that is e.g. 1.002356456867  On guisettings.xml is stored with more decimals and you can see the problem. Also on BD or debugging the code.

Reply:
With round code the same number shown on screen is stored on xml.  

With NO round code the number saved on xml is not the same as shown on screen.

Reply:
That may be a larger issue we need to solve then, but I don't want to hold this PR up anymore. I'd rather remove the rounding code and we can look at it later.

Reply:
Done